packages/react-sdk/src/components/meta.ts

```diff
@@ -58,6 +58,6 @@ type MetaProp =
 
 type MetaProps = Record<string, MetaProp>;
 
-// Probably better instead of JSON to generate ts with `as const`
```

Comment:
Deleted this code comment by mistake. Still applies

Reply:
https://github.com/webstudio-is/webstudio-designer/issues/521